tests/acceptance/conftest.py

```diff
@@ -75,33 +73,14 @@ def app(request):
 
 @pytest.fixture
 def arsenic(selenium, app):
-    return Arsenic(selenium)
-
-
-@pytest.fixture(scope='session', autouse=True)
-def cleanup_cookies():
-    cookies_path = './tests/acceptance/login_cookies.pkl'
-    if os.path.exists(cookies_path):
-        os.unlink(cookies_path)
+    Arsenic(selenium)
```

Comment:
I think that the return was ok, even if it's not used, it will give you what you expect (even if it's for debugging). Returning `None` is useless.